### ISSUE TITLE
Display on top of fullscreen windows

### ISSUE THREAD

**Issue by lord-re** [muevoid/Wayherb#4]
Would it be possible to be displayed on top everything even fullscreen windows ?

**Comment by muevoid** [comment]
I don't believe so but I may be wrong... its been a while since ive used wayherb personally so I haven't worked on it much at all. If anyone provides resources to examples of the layer protocol drawing over fullscreen windows though i can try to implement it

**Comment by notthewave** [comment]
Should be possible with layershell

**Comment by Vixeliz** [comment]
We actually are already using layershell set to top. I just dont think it appears over fullscreen windows but i may be wrong. Also @notthewave I saw your fork if youd like to make pull request here feel free

**Comment by Vixeliz** [comment]
Also alpha is already supported btw

**Comment by notthewave** [comment]
I actually wanted to start learning C with wayherb but IDK where to start. So I didn't do anything yet.

**Comment by Vixeliz** [comment]
Alpha is optional you can just set the values to 1.0 for solid. As for learning C with wayherb this code isnt the cleanest to be honest. If you are interested in game development at all or even simple gui applications check out raylib and raygui it is a great library to learn some basics of c with a great community.
